vMAC guides update - Scale/HGR availability (#7859)
ovh/docs: pages/bare_metal_cloud/dedicated_servers/network_bridging/guide.en-gb.md

---

> title: 'Configuring a network bridge'

Reviewer: Pas lié à cette PR mais je trouve le titre et la description un peu étranges. Il faudrait mentionner "Additional IP" quelque part, non ?

Author: Effectivement, ça paraît incomplet. Le titre en français est d'autant plus étrange. Je fais la modif.

> title: 'Configuring Additional IP in bridge mode on your virtual machines'

Reviewer: ```suggestion
title: 'Configuring Additional IPs in bridge mode on your virtual machines'
```

>
> Please visit our [comparison page](/links/bare-metal/eco-compare) for more information.
>

> > As of May 2025, this guide can be used for servers of the ranges [Scale](https://www.ovhcloud.com/en-gb/bare-metal/scale/) and [High Grade](https://www.ovhcloud.com/en-gb/bare-metal/high-grade/).

Reviewer: ```suggestion
> As of May 2025, this guide can be used for servers of the [Scale](https://www.ovhcloud.com/en-gb/bare-metal/scale/) and [High Grade](https://www.ovhcloud.com/en-gb/bare-metal/high-grade/) ranges.
```

>

> > Alternative methods are available to configure the network on your virtual machines hosted on servers from those ranges : [Configuring the network on Proxmox VE on the High Grade & Scale ranges](/pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale) and [Configuring the network on Windows Server with Hyper-V on the High Grade & Scale ranges](/pages/bare_metal_cloud/dedicated_servers/hyperv-network-HG-Scale).

Reviewer: ```suggestion
> Alternatively, to configure Additional IPs using in routed mode or in a vRack, please refer to [Configuring the network on Proxmox VE on the High Grade & Scale ranges](/pages/bare_metal_cloud/dedicated_servers/proxmox-network-HG-Scale) or [Configuring the network on Windows Server with Hyper-V on the High Grade & Scale ranges](/pages/bare_metal_cloud/dedicated_servers/hyperv-network-HG-Scale).
```
